### PR TITLE
DAMIEN-67: API for editing a department contact

### PR DESCRIPTION
**Pull request by lyttam**
https://jira-secure.berkeley.edu/browse/DAMIEN-67

### CODE REVIEW

**Comment by lyttam** [comment]
@pauline2k @johncrossman This PR would like a review please.

File: damien/api/department_controller.py [hunk 1]
```diff
@@ -71,3 +73,34 @@ def update(department_id):
         return tolerant_jsonify(department.to_api_json())
     else:
         raise ResourceNotFoundError(f'Department {department_id} not found.')
+
+
+@app.route('/api/department/<department_id>/contact', methods=['POST'])
+@login_required
+def update_contact(department_id):
+    if not current_user.is_admin:
+        raise ForbiddenRequestError('Admin required.')
```
Comment (johncrossman) [inline damien/api/department_controller.py:82]: @lyttam - would a `@admin_required` annotation make sense

Comment (pauline2k) [reply damien/api/department_controller.py:82]: Oh yeah, we probably are hitting the tipping point of that decorator being useful.

Comment (lyttam) [reply damien/api/department_controller.py:82]: @johncrossman Yes, good catch. I will change it in the next PR.

**Pull request merged**